### ISSUE TITLE
About implements details

### ISSUE THREAD

**Issue by sasa550** [ChenHongruixuan/MambaCD#84]
Hello authors,
I'm confused with the metric calculation in semantic change detection. From the output, I observed that during the train() process, the unique values of output_semantic_t1 after the softmax calculation range from (0-6). However, during the validation() process, the unique values range from (1-5). How does this difference arise?

**Comment by ChenHongruixuan** [comment]
Hi @sasa550 ,

Thank you for your question! We did not check this. However, I think the question you raise depends on the division of the training set/test set and the category situation of the samples in it. Regarding this, we use the officially divided training set/test set.

Best,
